### ISSUE TITLE
no-unsupported-features/node-builtin should detect unsupported usage of `REPLServer#setupHistory`

### ISSUE THREAD

**Issue by Skn0tt** [mysticatea/eslint-plugin-node#222]
The following is supported from Node v12 onwards:

```js
const REPL = require("repl");
const repl = REPL.start();
repl.setupHistory(...);
```

It'd be awesome if no-unsupported-features/node-builtin caught this.

For reference: blitz-js/blitz#454
